stores/stripe.ts

```diff
@@ -0,0 +1,19 @@
+import { defineStore } from 'pinia'
+import { LIKE_CO_API } from '~/constant'
+
+export const useStripeStore = defineStore('stripe-connect', () => {
+  const stripeConnectStatusWalletMap = ref({} as Record<string, any>)
+
+  async function fetchStripeConnectStatus (wallet: string) {
+    stripeConnectStatusWalletMap.value[wallet] = { isReady: false }
+    const connectStatusData = await useFetch(
+        `${LIKE_CO_API}/likernft/book/user/connect/status?wallet=${wallet}`
+    )
```

Comment:
Should throw if `stripeConnectStatusWalletMap.error` is trueful

Reply:
[🎨 Improve error handling](https://github.com/likecoin/nft-book-press/pull/139/commits/dc3b1af80b3dfcbd2f31238d39c2d9c31dc5df47)